code/modules/cargo/packs/security.dm

```diff
@@ -25,6 +25,14 @@
 	cost = 2500
 	contains = list(/obj/effect/spawner/lootdrop/f13/bomb/tier2)
 
+/datum/supply_pack/security/attachmentsf13
```

Comment:
Add gun parts to this as well, since most semi-auto pistols can be modified now with a screwdriver and crowbar to swap out the internals.